client/src/main/scala/org/zalando/nakadi/client/scala/ClientImpl.scala

```diff
@@ -178,7 +181,7 @@ class ClientImpl(connection: Connection, subscriber: SubscriptionHandler, charSe
       case HttpResponse(status, headers, entity, protocol) if (status.isSuccess()) =>
         try {
           Unmarshal(entity).to[String].map { body =>
-            logger.debug(s"Payload: $body")
+            logger.info(s"Payload: $body")
```

Comment:
- actually, the former logging level was more suitable
- please change to logger.debug("Payload: {}", body)
  -> otherwise, a new string is always constructed even though the logging level was turned of